code/game/objects/structures/crates_lockers/closets/secure/medical.dm

```diff
@@ -275,3 +267,29 @@
 		new /obj/item/weapon/reagent_containers/glass/bottle/reagent/acetone(src)
 		new /obj/item/weapon/reagent_containers/glass/bottle/reagent/acid(src)
 		new /obj/item/weapon/reagent_containers/glass/bottle/reagent/diethylamine(src)
+
+
+/obj/structure/closet/secure_closet/coroner
+	name = "coroner's locker"
+	icon_state = "securecoroner1"
+	icon_closed = "securecoroner"
+	icon_locked = "securecoroner1"
+	icon_opened = "securecoroneropen"
+	icon_broken = "securecoronerbroken"
+	icon_off = "securecoroneroff"
+	req_access = list(access_morgue)
+
+	New()
+		..()
+		if(prob(50))

```

Comment:
Can I ask why you've done this out of interest instead of just adding both?

Reply:
It follows the same scheme as many other lockers (except, research for some reason). The locker always contain a duffel bag and a chance it would spawn 1 of the 2 normal bags. I figured if the Coroner has to haul things around when it comes to personal items of the dead corpse or whatever, I personally think it is a good idea to make it follow the same format.

Its still open for change.